ci/setup-build-test-gdk.ps1

```diff
@@ -34,61 +50,56 @@ class TestSuite {
     }
 }
 
-[string] $test_repo_url = "git@github.com:improbable/UnrealGDKEngineNetTest.git"
-[string] $test_repo_relative_uproject_path = "Game\EngineNetTest.uproject"
-[string] $test_project_name = "NetworkTestProject"
-[string] $test_repo_branch = "master"
 [string] $user_gdk_settings = "$env:GDK_SETTINGS"
 [string] $user_cmd_line_args = "$env:TEST_ARGS"
 [string] $gdk_branch = "$env:BUILDKITE_BRANCH"
 
-# If the testing repo has a branch with the same name as the current branch, use that
-$testing_repo_heads = git ls-remote --heads $test_repo_url $gdk_branch
-if($testing_repo_heads -Match [Regex]::Escape("refs/heads/$gdk_branch")) {
-    $test_repo_branch = $gdk_branch
-}
+[TestProjectTarget] $gdk_test_project = [TestProjectTarget]::new("git@github.com:spatialos/UnrealGDKTestGyms.git", $gdk_branch, "Game\GDKTestGyms.uproject", "GDKTestGyms")
+[TestProjectTarget] $native_test_project = [TestProjectTarget]::new("git@github.com:improbable/UnrealGDKEngineNetTest.git", $gdk_branch, "Game\EngineNetTest.uproject", "NativeNetworkTestProject")
 
 # Allow overriding testing branch via environment variable
-if (Test-Path env:TEST_REPO_BRANCH) {
-    $test_repo_branch = $env:TEST_REPO_BRANCH
+if (Test-Path env:GDK_TEST_REPO_BRANCH) {
+    $gdk_test_project.test_repo_branch = $env:GDK_TEST_REPO_BRANCH
```

Comment:
Hmm, unfortunately `TEST_REPO_BRANCH` is referenced in docs, and is used by other pipelines.
I would prefer to keep it intact.

Reply:
Changed back!